recipes/chamois/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "CHAMOIS" %}
+{% set name_pypi = "chamois-tool" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.org/packages/source/c/{{ name_pypi }}/{{ name_pypi | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 4812d2209b3867236892197348ce9c6ddecf7e7444207d4ced10a2068d898b8e
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  run_exports:
+    - {{ pin_subpackage("chamois", max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools >=39.2
+    - wheel >=0.30
+  run:
+    - python >=3.7
+    - importlib_resources >=1.0  #[py37, py38, py39]
+    - numpy >=1.16
+    - anndata >=0.8
+    - gb-io >=0.3.1
+    - lz4 >=4.0
```

why not add `isal` (may need to be in the host section)
```suggestion
    - lz4 >=4.0,<5.dev0
```

---

I need the `python-lz4` bindings at runtime, not for compiling anything, so I don't think they need to be in the host section?